src/bootstrap/src/core/config/config.rs

```diff
@@ -2823,6 +2820,7 @@ impl Config {
 
 /// Compares the current `Llvm` options against those in the CI LLVM builder and detects any incompatible options.
 /// It does this by destructuring the `Llvm` instance to make sure every `Llvm` field is covered and not missing.
+#[cfg(not(test))]
```

Comment:
Why is this here? (and why doesn't it use `#[cfg(not(feature = "bootstrap-self-test"))]` instead? :thinking:).

Reply:
Both works actually. Changed it to "bootstrap-self-test".